script/releases/v1.0.0-slashing/1-deployContracts.s.sol

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {EOADeployer} from "zeus-templates/templates/EOADeployer.sol";
+import "../Env.sol";
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+/**
+ * Purpose: use an EOA to deploy all of the new contracts for this upgrade. 
+ */
+contract Deploy is EOADeployer {
+    using Env for *;
+
+    function _runAsEOA() internal override {
+        vm.startBroadcast();
+
+        /// permissions/
+
+        address[] memory pausers = new address[](3);
+        pausers[0] = Env.pauserMultisig();
+        pausers[1] = Env.opsMultisig();
+        pausers[2] = Env.executorMultisig();
+
+        deployImpl({
+            name: type(PauserRegistry).name,
+            deployedTo: address(new PauserRegistry({
+                _pausers: pausers,
+                _unpauser: Env.executorMultisig()
+            }))
+        });
+
+        deployImpl({
+            name: type(PermissionController).name,
+            deployedTo: address(new PermissionController())
+        });
+
+        deployProxy({
+            name: type(PermissionController).name,
+            deployedTo: address(new TransparentUpgradeableProxy({
+                _logic: address(Env.impl.permissionController()),
+                admin_: Env.proxyAdmin(),
+                _data: ""
+            }))
+        });
+
+        /// core/
+
+        deployImpl({
+            name: type(AllocationManager).name,
+            deployedTo: address(new AllocationManager({
+                _delegation: Env.proxy.delegationManager(),
+                _pauserRegistry: Env.impl.pauserRegistry(),
+                _permissionController: Env.proxy.permissionController(),
+                _DEALLOCATION_DELAY: Env.MIN_WITHDRAWAL_DELAY(),
```

Comment:
<img width="798" alt="image" src="https://github.com/user-attachments/assets/1295ac2c-7ad2-499b-a008-c8aba8a8efc0" />

These are returning different values from what we expect

Reply:
60 for withdrawal delay and 90 for allocation config delay: https://www.notion.so/eigen-labs/Slashing-Upgrade-14d13c11c3e080dca3abfb2eca7a8bf4?pvs=4#14d13c11c3e0803ca724c68d09638918

Reply:
Have fixed this in the metadata repo